flight/PiOS/STM32/pios_servo.c

```diff
@@ -683,6 +690,12 @@ static int DSHOT_Update()
 			case SYNC_DSHOT_600:
 			case SYNC_DSHOT_1200:
 				break;
+			case SYNC_DSHOT_DMA:
```

Comment:
Now we treat DMA/non-dma more consistently.  There's still duplicate code that can be eliminated.